### PR TITLE
chore(release): update changelog for v0.22.0 release

### PR DESCRIPTION
**Pull request by gabrielmer**
# Description

Changelog for v0.22.0

Please review carefully and suggest highlights, upgrade instructions or notes that should be added :)

# Changes

- [x] updated changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,67 @@
+## v0.22.0 (2023-11-15)
+
+> Note: The `--topic` option is now deprecated in favor of a more specific options `--pubsub-topic` & `--content-topic`
+
+> Note: The `ext-multiaddr-only` CLI flag was introduced for cases in which the user wants to manually set their announced addresses
```
Comment (NagyZoltanPeter) [inline CHANGELOG.md:5]: Maybe better referring to it as a switch if its that.
```suggestion
> Note: The `--ext-multiaddr-only` CLI flag was introduced for cases in which the user wants to manually set their announced addresses
```

**Review by NagyZoltanPeter — APPROVED** [review]
Little observation, but LGTM overall! Thank you.

**Review by SionoiS — APPROVED** [review]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (gabrielmer) [reply CHANGELOG.md:5]: Oh great catch! Thank you!

**Review by vpavlin — APPROVED** [review]

**Pull request merged**